wikimedia/wikipedia-ios
# [ALT TEXT] Add guidance presentation

**Phabricator:** https://phabricator.wikimedia.org/T370330

### Notes
This PR presents an adapted onboarding view after tapping the "Guidance" link in the alt text flow. In order to keep things a little cleaner, I refactored the previous onboarding presentations into these "presenter" classes:

- AltTextImageRecommendationsOnboardingPresenter - handles presenting alt text onboarding from the image recommendations flow (Flow B)
- AltTextArticleEditorOnboardingPresenter - handles presenting alt text onboarding from the article editor flow (Flow C)
- AltTextGuidancePresenter - handles presenting the guidance view (adapted onboarding) from within the alt text flow (Flows B and C).

Note this PR is dependent on #4929. That PR should be reviewed and merged first.

### Test Steps
1. (Optional) Maybe repeat the test steps in #4929 to confirm the refactor didn't break anything
2. Go into the alt text input view in flow B and tap the "Guidance" link. Confirm onboarding is presented and primary button says "Done".
2. Go into the alt text input view in flow C and tap the "Guidance" link. Confirm onboarding is presented and primary button says "Done".
